skbio/util/_testing.py

```diff
@@ -94,3 +94,12 @@ def get_data_path(fn, subfolder='data'):
     path = os.path.dirname(os.path.abspath(callers_filename))
     data_path = os.path.join(path, subfolder, fn)
     return data_path
+
+
+def _skip_if_no_matplotlib():
```

Comment:
This would be incredibly awesome if it was a decorator, then it acts as a bit of a "living comment". It could even be generalized to something like:

```
@skip_if_missing("matplotlib")
def test_something(self):
    pass
```
You can use [import_module](https://docs.python.org/3/library/importlib.html#importlib.import_module) to import via a string.